.devcontainer/cuda11.8-conda/devcontainer.json

```diff
@@ -8,6 +8,11 @@
       "BASE": "rapidsai/devcontainers:24.06-cpp-cuda11.8-mambaforge-ubuntu22.04"
     }
   },
+  "runArgs": [
+    "--rm",
+    "--name",
+    "rapids-${localWorkspaceFolderBasename}-24.06-cuda11.8-conda"
```

Comment:
The `ci/release/update-version.sh` file needs to be updated to cover this.

Reply:
Done